docs/introduction.rst

```diff
@@ -1,73 +1,57 @@
 
-In addition, the main window has a variety of utility panels attached or associated with the main window. Some examples of utility panels are the data browser panel, the inspector panel, and the info and histogram panels.
+Graphics and Masks
+==================
+Nion Swift provides plenty of useful graphics to narrow down your view of the data. Use graphics to highlight portions of the data, select areas for processing, or to focus on specific sections of data.
 
-See :ref:`user-interface`.
+Masking allows you to block out the rest of an image and see only what you want. Choose from a variety of mask shapes to precisely define your area of interest.
 
-Display Panels
---------------
-Workspaces are split into one or more display panels. Each display panel can show one display item.
+See :ref:`graphics` for more information.
 
-A typical display item would show a single data item, but you can create multiple display items for each data item to visualize the data differently at the same time.
+Processing and Analysis
+=======================
+Processing and analysis greatly expand the functionality of Nion Swift. The software provides many options for filtering, transforming, and more. Computations allow for specific and customizable processes, and there is even support for python scripting to add your own computations.
 
-In addition, some display items may display multiple data items at once: a line plot may have multiple layers, each displaying a different data item or even different reductions of the same data item.
+See :ref:`processing` for more information.
 
-See :ref:`display-panels`.
+File Management
+===============
+Make use of Nion Swift without live acquisition by importing your own data files into the program, take your files with you by exporting your data, save your project files and use them on any other installation of Nion Swift, make copies of your data to create a backup, and contact us about recovery options.
 
-Computations
-------------
-Computations are processing algorithms that take input data items and other parameters to produce output data items. The algorithm watches for changes to the input data items and automatically runs the processing when it sees a change.
-
-In this way, it is easy to create new data items that update in real time during acquisition or when the parameters change.
-
-See :ref:`processing`.
-
-Graphics
---------
-Graphics are annotations or other graphical items that can be attached to display items.
-
-Examples of graphics would be a point, line, or rectangle on a 2D image or a interval or channel on a 1D plot.
-
-Additionally, Fourier mask filter graphics, such as band pass filters or lattice filters, can be attached to data in the Fourier domain.
-
-See :ref:`graphics`.
+See :ref:`data-management` for more information.
```

Comment:
One additional note: the old overview included a section on "projects". I think we should still include that. Was there a reason you dropped it from this section? The reason I think it's important is to introduce users to the term "project" a.s.a.p. I'm open to other suggestions, but that's my thought.